services/QuillLMS/app/mailers/user_mailer.rb

```diff
@@ -40,6 +45,99 @@ def account_created_email(user, temp_password, admin_name)
+  end
+
+  def admin_dashboard_made_school_admin_email(user, admin_name, school_name)
+    @user = user
+    @admin_name = admin_name
+    @school_name = school_name
+    mail to: user.email, subject: "#{user.first_name}, you are now a Quill admin for #{school_name}"
+  end
+
+  def admin_dashboard_made_school_admin_link_school_email(user, admin_name, school)
+    @user = user
+    @admin_name = admin_name
+    @school = school
+    @link_school_link = "#{ENV['DEFAULT_URL']}/teachers/#{@user.id}/schools/#{@school.id}"
+    mail to: user.email, subject: "#{user.first_name}, you are now a Quill admin for #{school.name}"
+  end
+
+  def admin_dashboard_made_school_admin_change_school_email(user, admin_name, new_school, existing_school)
+    @user = user
+    @admin_name = admin_name
+    @new_school = new_school
+    @existing_school = existing_school
+    @link_school_link = "#{ENV['DEFAULT_URL']}/teachers/#{@user.id}/schools/#{@new_school.id}"
+    mail to: user.email, subject: "#{user.first_name}, you are now a Quill admin for #{new_school.name}"
+  end
+
+  def internal_tool_admin_account_created_email(user, school_name)
+    @user = user
+    @school_name = school_name
+    @set_password_link = ENV['DEFAULT_URL']
+    mail to: user.email, subject: "[Action Required] #{user.first_name}, a Quill school admin account was created for you"
+  end
+
+  def internal_tool_made_school_admin_email(user, school_name)
+    @user = user
+    @school_name = school_name
+    mail to: user.email, subject: "#{user.first_name}, you are now a Quill admin for #{school_name}"
+  end
+
+  def internal_tool_made_school_admin_link_school_email(user, school)
+    @user = user
+    @school = school
+    @link_school_link = "#{ENV['DEFAULT_URL']}/teachers/#{@user.id}/schools/#{@school.id}"
+    mail to: user.email, subject: "#{user.first_name}, you are now a Quill admin for #{school.name}"
+  end
+
+  def internal_tool_made_school_admin_change_school_email(user, new_school, existing_school)
+    @user = user
+    @new_school = new_school
+    @existing_school = existing_school
+    @link_school_link = "#{ENV['DEFAULT_URL']}/teachers/#{@user.id}/schools/#{@new_school.id}"
+    mail to: user.email, subject: "#{user.first_name}, you are now a Quill admin for #{new_school.name}"
+  end
+
+  def internal_tool_district_admin_account_created_email(user, district_name)
+    @user = user
+    @district_name = district_name
+    @set_password_link = ENV['DEFAULT_URL']
+    mail to: user.email, subject: "[Action Required] #{user.first_name}, a Quill district admin account was created for you"
+  end
+
+  def internal_tool_made_district_admin_email(user, district_name)
+    @user = user
+    @district_name = district_name
+    @set_password_link = ENV['DEFAULT_URL']
+    mail to: user.email, subject: "#{user.first_name}, you are now a Quill admin for #{district_name}"
+  end
+
   # Sent when an admin requests an existing teacher to join one of their schools.
```

Comment:
This looks like it's going to be a duplicate of one of the emails above, is this just gonna get cleaned up later?